notion-distcopy/src/test/scala/com/ambiata/notion/distcopy/DistCopySpec.scala

```diff
@@ -28,7 +28,7 @@ class DistCopySpec extends AwsScalaCheckSpec(tests = 5) { def is = s2"""
    the full path of the file to copy $createDownloadMappingForADirectory
 
   An upload mapping must contain
-   the full path of the file to copy $createUploadMappingForADirectory
+    the path of the file to copy relative to the from dir $createUploadMappingForADirectory
```

Comment:
This is on purpose. We don't want to have clashes when we sync files to HDFS. For example if we synchronize files from `s3://bucket/LYL/year=2015/*` and `s3://bucket/TRN/year=2015/*` we don't want downloaded files to be possibly overwritten, where as when we synchronize back all the "output" files coming from the sync directory will share the same prefix which is the sync directory path so should not overlap when being copied back to S3. This might not be very clear, so I agree an example is in order... (I had also noticed the asymmetry at first and now I think it makes sense).